starters/serverless-framework-sqs-dynamodb/src/handlers/technology_create.test.ts

```diff
@@ -0,0 +1,111 @@
+import type { APIGatewayProxyResult, APIGatewayProxyEvent, Context, Callback } from 'aws-lambda';
+import { PutItemCommand, ServiceInputTypes, ServiceOutputTypes } from '@aws-sdk/client-dynamodb';
+import { AwsStub, mockClient } from 'aws-sdk-client-mock';
+import { getClient } from '@/utils/dynamodb/getClient';
+import { removeFromCache } from '@/utils/cache/removeFromCache';
+import * as technologyCreate from '@/models/technology/create';
+import { getCacheKey } from '@/models/technology/getCacheKey';
+import { handler } from './technology_create';
+
+describe('POST /technology', () => {
+	let subject: APIGatewayProxyResult;
+	let ddbMock: AwsStub<ServiceInputTypes, ServiceOutputTypes>;
+
+	describe('when record is created', () => {
+		beforeAll(async () => {
+			ddbMock = mockClient(getClient());
+			ddbMock.on(PutItemCommand).resolves({
+				Attributes: undefined,
+			});
+			subject = (await handler(
+				{
+					body: JSON.stringify({
+						description:
+							'Jest is a delightful JavaScript Testing Framework with a focus on simplicity. It works with projects using: Babel, TypeScript, Node, React, Angular, Vue and more!',
+						websiteUrl: 'https://jestjs.io/',
+						displayName: 'Valid Test',
+					}),
+				} as unknown as APIGatewayProxyEvent,
+				{} as Context,
+				{} as Callback
+			)) as APIGatewayProxyResult;
+		});
+
+		afterAll(async () => {
+			ddbMock.restore();
+			await removeFromCache(getCacheKey('87af19b1-aa0d-4178-a30c-2fa8cd1f2cff'));
```

Comment:
Where do you set/hardcode this cache key when setting the item into the cache? 🤔 

I also think that this uuid string should be exported as a `MOCK_TECHNOLOGY_ID` constant and use across tests.

Reply:
This is actually just an oversight bug. Should be `subject.id` here.

Reply:
These were just bad copy/pastes from other tests I wrote. They aren't actually needed.